### PR TITLE
create dockerfile to run echo service

### PR DESCRIPTION
**Pull request by wlu2016**
endpoints alpha doc will need to link to this dockerfile

### CODE REVIEW

**Comment by jonparrott** [comment]
This will actually break the flexible samples, as the presence of a `Dockerfile` will cause `gcloud app deploy` to complain about `runtime` not being set to `custom`.

We have two options:
* We could put the Dockerfile elsewhere or by a different name and tell the user to copy it.
* we can duplicate this sample to `container_engine/endpoints`.

**Comment by wlu2016** [comment]
if I rename this to echo.Dockerfile, will this solve that problem?

By the way, like the idea of having a sample in container_engine/endpoints. But I am hesitating on causing code duplicacy. Let me know what you would prefer :)

**Comment by wlu2016** [comment]
they don't need to copy it. we document in the user doc and ask them to run:
docker build -t image_path ./echo.Dockerfile

**Comment by jonparrott** [comment]
I'd prefer `Dockerfile.container-engine`.

**Comment by wlu2016** [comment]
done. in the same directory? not sure if this will pass the CI.

By the way, this sample will be used for both gce and gke.

**Comment by jonparrott** [comment]
For the record @wlu2016 is a Googler and no CLA is needed.

File: managed_vms/endpoints/Dockerfile.container-engine [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM debian:jessie
+
+RUN apt-get update && \
+    apt-get install -y python2.7 python-pip python-gevent && \
```
Comment (jonparrott) [inline managed_vms/endpoints/Dockerfile.container-engine]: gevent shouldn't be needed.

File: managed_vms/endpoints/Dockerfile.container-engine [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    apt-get clean && \
+    rm /var/lib/apt/lists/*_*
+
+COPY requirements.txt /requirements.txt
```
Comment (jonparrott) [inline managed_vms/endpoints/Dockerfile.container-engine]: Why not just `ADD . /app` ?

**Comment by wlu2016** [comment]
done ptal

**Comment by jonparrott** [comment]
LGTM.

**Pull request merged**